app/javascript/pages/admin/Reporting/index.jsx

```diff
@@ -0,0 +1,254 @@
+
+  const reportKeyTitleMap = {
+    user: t('volunteer_portal.admin.tab.reporting.dropdown.users')
+  }
+
+  const handleDropdownStateChange = (changes, stateAndHelpers) => {
+    if (Object.prototype.hasOwnProperty.call(changes, 'isOpen')) {
+      const nextIsOpen =
+        changes.selectedItem === 'general' ||
+        changes.selectedItem === 'organized' ||
+        changes.selectedItem === 'individual' ||
+        changes.isOpen
+      setDropDownIsOpen(nextIsOpen)
+    }
+
+    if (Object.prototype.hasOwnProperty.call(changes, 'selectedItem')) {
+      const nextSelectedItem = changes.selectedItem
+
+      if (nextSelectedItem === 'general') {
+        switch (dropDownTempSelectedItem) {
+          case 'organized':
+            stateAndHelpers.setHighlightedIndex(1)
+            break
+          case 'individual':
+            stateAndHelpers.setHighlightedIndex(2)
+            break
+        }
+      }
+
+      setDropDownTempSelectedItem(nextSelectedItem)
+    }
+  }
+
+  const handleDropdownOnSelect = item => {
+    if (item !== 'general' && item !== 'individual'  && item !== 'organized') {
+      setReport(item)
+    }
+  }
+
+  const renderMenu = () => (
+    <Menu >
+      {dropDownTempSelectedItem === 'organized' ? (
+        <>
+          <PreviousItem value="general">{t('volunteer_portal.admin.tab.reporting.dropdown.organized_events')}</PreviousItem>
+          <Separator />
+          <Item value="organizedEventType">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.event_type')}</Item>
+          <Item value="organizedTag">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.tag')}</Item>
+          <Item value="organizedOrganization">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.organization')}</Item>
+          <Item value="organizedTotal">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.total')}</Item>
+        </>
+      ) : dropDownTempSelectedItem === 'individual' ? (
+        <>
+          <PreviousItem value="general">{t('volunteer_portal.admin.tab.reporting.dropdown.individual_events')}</PreviousItem>
+          <Separator />
+          <Item value="individualEvent-type">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.event_type')}</Item>
+          <Item value="individualTag">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.tag')}</Item>
+          <Item value="individualOrganization">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.organization')}</Item>
+          <Item value="individualTotal">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.total')}</Item>
+        </>
+      ) : (
+        <>
+          <Item value="user">{t('volunteer_portal.admin.tab.reporting.dropdown.users')}</Item>
+          <NextItem value="organized" disabled>{t('volunteer_portal.admin.tab.reporting.dropdown.organized_events')}</NextItem>
+          <NextItem value="individual" disabled>{t('volunteer_portal.admin.tab.reporting.dropdown.individual_events')}</NextItem>
```

Comment:
I don't know about this tbh.
1. From an end user perspective, It might feel like a bug. I don't think this communicates that it is an unimplemented feature.
2. I don't think unimplemented features have any business being in the UI. Usually disabled options means that it will be enabled when certain conditions are met (i.e. permissions or conditional data). 
3. There is a lot of complex switching logic to accomodate this unimplemented. This logic will run unnecessarily, adds surface area for bugs and makes it hard to reason about imo.

Reply:
That's a fair point, I'll remove this and add the options as they're implemented.